### PR TITLE
feat: make fonts customizable

### PR DESCRIPTION
**Pull request by v4vong**
## Description

Move the font family and font size of the body and headings into `variables.scss` so that making them be more easily customizable for users.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## How has this been tested

- [ ] I have run `bash ./tools/test` (at the root of the project) locally and passed
- [x] I have tested this feature in the browser

### Test Configuration

- Browser type & version:
- Operating system:
- Ruby version: 
- Bundler version: 
- Jekyll version:

### CODE REVIEW

File: _sass/addon/commons.scss [hunk 1]
```diff
@@ -34,7 +34,7 @@ body {
     env(safe-area-inset-bottom) env(safe-area-inset-left);
   color: var(--text-color);
   -webkit-font-smoothing: antialiased;
-  font-family: 'Source Sans Pro', 'Microsoft Yahei', sans-serif;
+  font-family: $body-font;
```
Comment (cotes2020) [inline _sass/addon/commons.scss:37]: Please change to a more descriptive variable name：

```suggestion
  font-family: $font-family-base;
```

File: _sass/addon/module.scss [hunk 2]
```diff
@@ -7,7 +7,7 @@
 %heading {
   color: var(--heading-color);
   font-weight: 400;
-  font-family: Lato, 'Microsoft Yahei', sans-serif;
+  font-family: $heading-font;
```
Comment (cotes2020) [inline _sass/addon/module.scss:10]: Update the variable name:

```suggestion
  font-family: $font-family-heading;
```

File: _sass/jekyll-theme-chirpy.scss
Comment (cotes2020) [inline _sass/jekyll-theme-chirpy.scss:1]: Revert the changes here

File: _sass/addon/variables.scss [hunk 3]
```diff
@@ -28,3 +28,14 @@ $bottom-min-height: 35rem !default;
 /* syntax highlight */
 
 $code-font-size: 0.85rem !default;
+
+/* fonts */
+
+$body-font: 'Source Sans Pro', 'Microsoft Yahei', sans-serif;
+$heading-font: Lato, 'Microsoft Yahei', sans-serif;
+$h1-size: 1.9rem;
```
Comment (cotes2020) [inline _sass/addon/variables.scss:36]: 1. Rename the variables for font-family, what's more
2. Drop the definition of heading font sizes, as they will be produced by a formula in the future. 
3. Finally, please take care of the CI results and make improvements, thanks!

**Review by cotes2020 — CHANGES_REQUESTED** [review]

**Comment by v4vong** [comment]
Suggestions accepted.

File: _sass/jekyll-theme-chirpy.scss
Comment (v4vong) [reply _sass/jekyll-theme-chirpy.scss:1]: `variables` must be imported before `module` since `$font-family-heading` is referred to in `module`.

Comment (cotes2020) [inline _sass/jekyll-theme-chirpy.scss:1]: Almost ready for merging, and the modifications in `_sass/jekyll-theme-chirpy.scss` also need to be restored.

**Review by cotes2020 — CHANGES_REQUESTED** [review]

File: _sass/jekyll-theme-chirpy.scss
Comment (cotes2020) [reply _sass/jekyll-theme-chirpy.scss:1]: FYIP,  when making changes in PR, you can use `force push` to reduce unnecessary commits.

Comment (v4vong) [reply _sass/jekyll-theme-chirpy.scss:1]: > Almost ready for merging, and the modifications in `_sass/jekyll-theme-chirpy.scss` also need to be restored.

It doesn't need to be restored. `variables.scss` must be imported before `module.scss` since `%heading` in `module.scss` needs to refer to `$font-family-heading`.

Comment (v4vong) [reply _sass/jekyll-theme-chirpy.scss:1]: > FYIP, when making changes in PR, you can use `force push` to reduce unnecessary commits.
> 
> And [keeping your pull request in sync with the base branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/keeping-your-pull-request-in-sync-with-the-base-branch).

Thanks. Squashed them.

Comment (cotes2020) [reply _sass/jekyll-theme-chirpy.scss:1]: Can you [keep the pull request in sync with the base branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/keeping-your-pull-request-in-sync-with-the-base-branch)?

Comment (v4vong) [reply _sass/jekyll-theme-chirpy.scss:1]: Done
